framework/src/modules/chain/modules/delegates.js

```diff
@@ -899,6 +896,23 @@ Delegates.prototype.getForgersKeyPairs = function() {
 	return __private.keypairs;
 };
 
+Delegates.prototype.getForgingStatusForAllDelegates = function() {
```

Comment:
Why is it included in this PR?

Reply:
Because old code was using `delegate.list` in the `http_api` module that is not available there any more. So we have do this calculation on the chain module side.